### ISSUE TITLE
Terminal().location context manager emits nothing for 'ansi' and 'ansi-bbs' TERM types

### ISSUE THREAD

**Issue by haliphax** [jquast/blessed#44]

**Comment by haliphax** [comment]
checked with `TERM=ansi script -c "telnet localhost"`

**Comment by jquast** [comment]
restore cursor not in termcap,
```
In [1]: from blessed import Terminal

In [2]: Terminal('ansi').rc
Out[2]: ''
```

**Comment by jquast** [comment]
Closed by 938e1362592dcd356ed84f136b9a0afb96a22e2f
